Formula/libvmaf.rb

```diff
@@ -24,6 +25,7 @@ def install
       system "meson", *std_meson_args, "build"
       system "ninja", "-vC", "build"
       system "ninja", "-vC", "build", "install"
+      share.install "model"
```

Comment:
Would `pkgshare` be more appropriate, so it gets installed in `share/libvmaf/model` rather than a somewhat cryptic `share/model`?

Reply:
I think this is a reasonable adjustment. Especially since we can override the default path anyhow.